### DISCUSSION TITLE
How can I check which formulas I have updated and which formulas have been updated recently in tap?

### DISCUSSION THREAD

**Discussion by edoger** [Homebrew/discussions#2663] [Getting started]
I want to know which programs I have updated recently on my computer, and I also want to know which new programs are coming! I did not find any related commands~

**Comment by SMillerDev** [comment]
The first one is not something that is possible. For the second one it depends how the tap works. For homebrew core you can have a look at https://github.com/Homebrew/homebrew-core/pulls to see what changes are proposed by the community.

**Reply by edoger** [reply]
@SMillerDev Why can brew update sometimes tell me which software has been updated recently, but sometimes it can’t? How can I ensure that this command tells me which updates have recently occurred every time, I feel this is obtained from the git repository history , Why don’t we keep this command always outputting the latest software updates? What do you think?

**Reply by SMillerDev** [reply]
Because it will only tell you what changed since the list time you ran it.

If nothing changed it outputs nothing.

**Reply by edoger** [reply]
Understand, I will try to implement a ```brew history``` command to meet these requirements.

**Reply by SMillerDev** [reply]
If you want to know the full history of a tap, using the github commit RSS features seem like an easier solution.

**Comment by synkathairo** [comment]
`brew outdated` will show which versions of local formulas are outdated
